lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -235,6 +244,7 @@ class AwsCompileEventBridgeEvents {
         FunctionName,
         EventBridgeConfig: {
           RuleName,
+          State,
```

Comment:
Did you verify that it works for custom-resource-based implementation? I believe it's not going to be honoured at all

Reply:
Cool will take a look at updating that too, thanks :)

Reply:
@pgrzesik Have updated the custom-resource implementation, but haven't run any tests.